python/metatensor_core/metatensor/_html_stylesheet.py

```diff
@@ -0,0 +1,167 @@
+# CSS styles for the HTML representation
```

Comment:
This should not be in a separate file if we want to make it easy to use from C++ as well